app/assets/stylesheets/application.scss

```diff
@@ -213,3 +213,7 @@ body {
 .subject-group-credits-unmet {
   color: red !important;
 }
+
+#butInstall {
+  padding-left: 0px;
+}
```

Comment:
This feels pretty hacky: I had to do it because otherwise the space between the credits counter and the cloud icon will be different than the one for the cloud icon and the search bar:

<img width="375" alt="image" src="https://github.com/cedarcode/mi_carrera/assets/46354312/7b8f251f-5cf2-4d90-bb9e-ad945b03e8b3">

by setting the `padding-left` to be `0`, the icon is centered in the middle so the space between the three elements looks the same – although I'm not entirely sure is _exactly_ the same:

<img width="374" alt="image" src="https://github.com/cedarcode/mi_carrera/assets/46354312/76d457fb-9a2a-4558-81ef-07df683ad515">

I think this is okay for now but we should do something with the space between the icons and credits counter in the nav bar.